### ISSUE TITLE
Increase Semantics in Main Subjects

### ISSUE THREAD

**Issue by LibrErli** [DieDatenlaube/DieDatenlaube#10]
We made a lot of subject cataloging for Gartenlaub articles, and we did it in a very classical subject-librarian-style. That means we conjunct different subject terms in the statement P921. 

But Wikidata allows us to setup semantic conjunctions in our main subject statements by using qualifiers:

1. We talked about dates and events in articles - which could be modelled through qualifiers https://github.com/DieDatenlaube/DieDatenlaube/issues/8
2. See the following item which is about sales of liquor in the Northern Sea which has an international treaty as result - i made a try to modell this in one statement: https://www.wikidata.org/wiki/Q81897431

**Comment by merfurth** [comment]
Yes, it would show the power of linked data in Wikidata. But therefore we should offer some more (query) examples to maintain these qualifiers, like mentioned under https://www.wikidata.org/wiki/Help:Qualifiers 

And for the count of our main subjects we need an machine based approach, to find these qualifiers.

**Comment by jeb140** [comment]
Yeah!  ... please send more examples to motivate the main subject-crowd

**Comment by LibrErli** [comment]
List all Articles with at least one main subject with qualifiers:

```
SELECT DISTINCT ?Die_Gartenlaube ?Die_GartenlaubeLabel ?QualSchlagwortLabel ?qualifierLabel ?schlagwortqualifierLabel WHERE {
  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
  hint:Query hint:optimizer "None".
  ?Die_Gartenlaube wdt:P1433 wd:Q655617;
    wdt:P921 ?schlagwort;
    p:P921 ?schlagwortstatement.
  ?schlagwortstatement ps:P921 ?QualSchlagwort;
    rdf:type wikibase:BestRank;
    ?qualifier ?schlagwortqualifier.
  ?qualifierproperty wikibase:qualifier ?qualifier.
}
ORDER BY (?Die_GartenlaubeLabel) (?QualSchlagwortLabel)
```
https://w.wiki/FT4

**Comment by jeb140** [comment]
a question might be: Which are the most useful cases to add a qualifier?

**Comment by merfurth** [comment]
One good reason was giving above in the help URL: For disputed items & community consensus ;)

Here are some good qualifiers we could use, to add further information to given subject: item _of_, _named as_ for proper nouns, _start/ end times_ of events, _stated in/as_ for source references and _connects with/ followed by_ to make relationships visible ...

**Comment by jeb140** [comment]
'to make relationships visible' is a point! As I concentrate on 'subject cataloging' for every item one time at least I see (for me) a trade off. I added yet qualifiers if I find variations between old and modern spelling. 

A more qualifier-based approach could be additionaly used for specific topics and series to get a semantic base for specific queries.
